test-engine-core/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "test-engine-core"
-version = "0.9.0"
```

Comment:
I think this version can remain at 0.9.0, most likely versioning is done in another way?